### ISSUE TITLE
How to support space in search text?

### ISSUE THREAD

**Issue by hawkerm** [mrlacey/CommentLinks#5]
Doesn't seem like there's a way with quoting or '%20' to add spacing to search text of a link.

i.e.
```cs
// link:MyFile.cs:"DependencyProperty SelectedTextProperty"
```

or
```cs
// link:MyFile.cs#:~:text=DependencyProperty%20SelectedTextProperty
```

**Comment by mrlacey** [comment]
xref #2

**Comment by mrlacey** [comment]
This functionality is included in v1.1 which is now available.
